resolwe_bio/processes/workflows/quantseq.py

```diff
@@ -0,0 +1,347 @@
+"""3' mRNA-Seq pipeline comprised of QC, preprocessing, alignment and quantification."""
+
+from resolwe.process import (
+    BooleanField,
+    Data,
+    DataField,
+    FloatField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    StringField,
+)
+from resolwe.process.models import Process as BioProcess
+
+
+class WorkflowQuantSeq(Process):
+    """3' mRNA-Seq pipeline.
+
+    Reads are preprocessed by __BBDuk__ or __Cutadapt__ which remove adapters,
+    trims reads for quality from the 3'-end, and discards reads that are too
+    short after trimming. Preprocessed reads are aligned by __STAR__
+    aligner. For read-count quantification, the __FeatureCounts__ tool
+    is used. QoRTs QC and Samtools idxstats tools are used to report
+    alignment QC metrics.
+
+    QC steps include downsampling, QoRTs QC analysis and alignment of
+    input reads to the rRNA/globin reference sequences. The reported
+    alignment rate is used to asses the rRNA/globin sequence depletion
+    rate.
+    """
+
+    slug = "workflow-quantseq"
+    name = "QuantSeq workflow"
+    requirements = {
+        "expression-engine": "jinja",
+    }
+    data_name = "{{ reads|sample_name|default('?') }}"
+    version = "5.0.0"
+    process_type = "data:workflow:quant:featurecounts"
+    category = "Pipeline"
+
+    class Input:
+        """Input fields."""
+
+        reads = DataField(
+            "reads:fastq",
+            label="Input reads (FASTQ)",
+            description="Reads in FASTQ file, single or paired end.",
+        )
+        genome = DataField(
+            "index:star",
+            label="Indexed reference genome",
+            description="Genome index prepared by STAR aligner indexing tool.",
+        )
+        adapters = ListField(
+            DataField("seq:nucleotide"),
+            label="Adapters",
+            description="Provide a list of sequencing adapters files (.fasta) "
```

Comment:
Expand the description, explaining the behaviour if the `adapters` input is not set (==Cutadapt trimming procedure is triggered). Even better would be to have an input to set the trimming tool explicitly (by using choices menu), which would then define the show/hide rules for the input groups based on the user choice.